src/screens/MovieCollection/MovieCollection.test.js

```diff
@@ -0,0 +1,71 @@
+import React from 'react';
+import { shallow } from 'enzyme';
+import renderer from 'react-test-renderer';
+import { render } from '@testing-library/react-native';
+import { withProviders } from '@/test-utils';
+import { MovieCollection } from '@/screens';
+import { movieList, navigation } from '@/test-utils/helper';
+import { MovieItem } from '@/components';
+
+const emptyRoute = { params: { collection: [{ movieList: [], title: 'test empty' }] } };
+const routeWithData = {
```

Comment:
are this names correct? or should they be inverted?

Reply:
The names were right, the data they contain is wrong 😂

Reply:
😂